docs/adrs/0003-bulk-import-ui.md

```diff
@@ -0,0 +1,98 @@
+# Bulk Import UI
+
+Date: 2020-10-26
+
+A primary goal of the current data upload refactoring project includes enabling users to upload data in bulk and producing multiple new data objects with only a few user-side operations. That means introducing an interface that can handle the following steps:
+* Selecting several data files to concurrently import into new data objects.
+* Launching what would amount to several import jobs at once (i.e. a bulk import operation).
+* Monitoring the state of those jobs and restarting if necessary.
+* Viewing the results of those jobs.
+
+Currently, this work is done by users using one cell at a time. An App Cell is generated for each desired data file upload, which can be time-consuming and cluttering for Narratives. The new design to support bulk operations.
```

Comment:
Nitpicky, but `The new design supports bulk operations`?

Reply:
fixed